avr/cores/megacommand/make_and_upload.sh

```diff
@@ -6,3 +6,6 @@ if [ $? -eq 0 ]; then
   echo RAM USAGE: $(${AVR_DIR}/bin/avr-size main.elf | grep main | awk '{ print $2 + $3}')
   ${AVR_DIR}/bin/avrdude -C${AVR_DIR}/etc/avrdude.conf -v -V -patmega2560 -cwiring -P${DEV} -b115200 -D -Uflash:w:./main.hex
 fi
+
+make -j8
```

Comment:
redundant.

Reply:
??

Reply:
Just marking a few redundant lines from the previous rebase. I'll clean it up when I have time.